src/main/kotlin/org/onflow/flow/sdk/models.kt

```diff
@@ -688,18 +467,7 @@ data class FlowChunk(
     val stateDeltaCommitment: ByteArray,
 ) : Serializable {
     companion object {
-        fun of(grpcExecutionResult: ExecutionResultOuterClass.Chunk) = FlowChunk(
-            collectionIndex = grpcExecutionResult.collectionIndex,
-            startState = grpcExecutionResult.startState.toByteArray(),
-            eventCollection = grpcExecutionResult.eventCollection.toByteArray(),
-            blockId = FlowId.of(grpcExecutionResult.blockId.toByteArray()),
-            totalComputationUsed = grpcExecutionResult.totalComputationUsed,
-            numberOfTransactions = grpcExecutionResult.numberOfTransactions,
-            index = grpcExecutionResult.index,
-            endState = grpcExecutionResult.endState.toByteArray(),
-            executionDataId = FlowId.of(grpcExecutionResult.executionDataId.toByteArray()),
-            stateDeltaCommitment = grpcExecutionResult.stateDeltaCommitment.toByteArray()
-        )
+        fun of(grpcExecutionResult: ExecutionResultOuterClass.Chunk) = FlowChunk(collectionIndex = grpcExecutionResult.collectionIndex, startState = grpcExecutionResult.startState.toByteArray(), eventCollection = grpcExecutionResult.eventCollection.toByteArray(), blockId = FlowId.of(grpcExecutionResult.blockId.toByteArray()), totalComputationUsed = grpcExecutionResult.totalComputationUsed, numberOfTransactions = grpcExecutionResult.numberOfTransactions, index = grpcExecutionResult.index, endState = grpcExecutionResult.endState.toByteArray(), executionDataId = FlowId.of(grpcExecutionResult.executionDataId.toByteArray()), stateDeltaCommitment = grpcExecutionResult.stateDeltaCommitment.toByteArray())
```

Comment:
This is really hard to read, can we keep this how it was?

Reply:
Will do @jribbink , the linter was failing builds because of the formatting so I had to refactor the file... let me see if I can revert it or override the rule here. Thanks!

Reply:
@jribbink reverted indentation back to the old format - should be good to go now. Sorry about that!